gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/PreserveAttributes.java

```diff
@@ -45,6 +45,7 @@ public static enum Option {
     GROUP('g'),
     PERMISSION('p'),
     VERSION('v'),
+    ACL('a'),
```

Comment:
nit: shouldn't this be plural, `ACLS`, since it preserves a list of them?

Reply:
I have updated the file with the above change in my latest commit